### ISSUE TITLE
Live render not decoding url parameters properly

### ISSUE THREAD

**Issue by benwilson512** [phoenixframework/phoenix_live_view#579]
For a URL path like:

```
/path/Wm9uZTozNzYxOA%3D%3D
```

The static render provides:
```
%{"id" => "Wm9uZTozNzYxOA=="}
```
but the live render provides:
```
%{"id" => "Wm9uZTozNzYxOA%3D%3D"}
```

@josevalim 

LiveView master@959b8f0c7fae91d41d79199458544fcbaaf5fb3d

**Comment by benwilson512** [comment]
Thanks!

**Comment by chrismccord** [comment]
Thanks for the report! ❤️❤️❤️🐥🔥

**Comment by cichacz** [comment]
Hello. This broke query string params. They're now being decoded twice, which causes bad behaviour for chars like `&=?%`. For some they just disappear (like `&`), and others (like `%`) just break the app.

Check this flow:
`phoenix_live_view/channel.ex:560 ... 570` -> `Utils.live_link_info!` -> `phoenix_live_view/utils.ex:176`

@chrismccord

**Comment by josevalim** [comment]
Hi @cichacz, can you please provide a concrete example of a URL that you are sending in the browser and it is not behaving as expected?

**Comment by cichacz** [comment]
Sure, so if you put something like this in url:
`http://localhost:4000/search?query=sometext+15%26`
then in `handle_params` the `params` will be `%{query: "sometext 15"}` where it should be "sometext 15&"

and visiting `http://localhost:4000/search?query=sometext+15%25`
will throw this:
```
[error] GenServer #PID<0.1090.0> terminating
** (Plug.Conn.InvalidQueryError) invalid urlencoded params, got sometext+15%
    (plug) lib/plug/conn/query.ex:96: Plug.Conn.Query.decode_www_form/2
    (plug) lib/plug/conn/query.ex:82: Plug.Conn.Query.decode_www_pair/3
    (elixir) lib/enum.ex:1948: Enum."-reduce/3-lists^foldl/2-0-"/3
    (phoenix_live_view) lib/phoenix_live_view/utils.ex:132: Phoenix.LiveView.Utils.live_link_info!/3
    (phoenix_live_view) lib/phoenix_live_view/channel.ex:546: Phoenix.LiveView.Channel.verified_mount/4
    (phoenix_live_view) lib/phoenix_live_view/channel.ex:34: Phoenix.LiveView.Channel.handle_info/2
    (stdlib) gen_server.erl:637: :gen_server.try_dispatch/4
    (stdlib) gen_server.erl:711: :gen_server.handle_msg/6
    (stdlib) proc_lib.erl:249: :proc_lib.init_p_do_apply/3
Last message: {:mount, Phoenix.LiveView.Channel}
```

**Comment by josevalim** [comment]
Thanks, fixed on master with eb717531d6379ec6426ebaf3249154d99b2868dc.

**Comment by cichacz** [comment]
Now it makes sense :) Thanks @josevalim !
